### ISSUE TITLE
Level: "The Mothership"

### ISSUE THREAD

**Issue by tttppp** [andybalaam/rabbit-escape#56]
I think I plugged some of the methods from UFO (#53). This is getting a bit on the large size though!
```
:name=The Mothership
:description=
:num_rabbits=20
:num_to_save=18
:dig=1
:bridge=3
:explode=3
:climb=2
:rabbit_delay=4
    #            #        #
          #          #     
 #    #        #           
                           
                           
                           
                           
                           
      /     /#\     \      
     /   /### ###\   \     
    /   (    #    )   \    
  )/ / / ##\ O /## \ \ \(  
 #()(# # ######### # #)()# 
  #\ # # #Q#####Q# # # /#  
   #\ \  #   #   #  / /#   
    #\ \ ### # ### / /#    
    /#\     /#\     /#\    
   /# #\ /#\   /#\ /# #\   
  /## ############### ##\  
```

**Comment by colonelfazackerley** [comment]
I have tried for 20 min or so. I have not been able to do it. I can get half of them out...

**Comment by colonelfazackerley** [comment]
The second time I came to this, I did it in 10 min. I managed to rescue all 20. I guess 2 of them are supposed to die, but I can't see a solution where that is a necessity.

It, like it's relative UFO, is an excellent level. Mothership is trickier. They should both be pushed to outdoors at the earliest convenience.

**Comment by tttppp** [comment]
No idea how you saved all 20!  I can save 19 using my method.  If your methods for the two levels are different from each other then I agree this should go in (possibly requiring to save 20 rather than 18).

**Comment by colonelfazackerley** [comment]
I think it's hard enough. I would keep the criterion at 18.

Let me know if you want me to post a screenshot of my solution.

**Comment by colonelfazackerley** [comment]
![mothership_soln](https://cloud.githubusercontent.com/assets/9596719/10412717/9b542ebe-6f86-11e5-93d2-808dca12cf87.png)

Here's my solution. I dropped a mini bridge stack in the pipe to get them all going clockwise.

I am curious what the intended solution is.

**Comment by tttppp** [comment]
This is the intended solution. I was foiled by bridge stacking again!

https://cloud.githubusercontent.com/assets/7073263/10412842/8beb5746-6f8a-11e5-9be2-bd3a8fe81c56.png

Saving 19 rather than 18 depends on using the second to last rabbit.

**Comment by colonelfazackerley** [comment]
Oh, that's very good. It never occurred to me that those blocks were anything but decorative.

**Comment by colonelfazackerley** [comment]
close me
merged to master with f96dd5696bc0d3c0f44b4856ba75e3a7d2bc201a

**Comment by andybalaam** [comment]
I did this on about the fifth try but I had accidentally read @colonelfazarkerley 's comment above. Great level. We may have to live with the bridge stacking solution unless we can think of a way around it.

**Comment by andybalaam** [comment]
First hint contains a backslash,  so reopening to fix that.

**Comment by colonelfazackerley** [comment]
I am noting your findings here
https://github.com/andybalaam/rabbit-escape/issues/88#issuecomment-158899114
i will do a pull request when you have got through the set. no sure it's worth opening each issue.

Sorry i spoilt it for you. I did wait 15 days. I have a couple of better methods now. I can post a link to an image or a solution.code.

**Comment by tttppp** [comment]
Would any of your other solutions work well as Deja Vu versions with fewer tokens available?

**Comment by colonelfazackerley** [comment]
The bridge stack takes 2 bridge tokens and 1 dig. Saves 20. Could use it in deja vu.

**Comment by andybalaam** [comment]
Deja Vu is #87

**Comment by andybalaam** [comment]
@colonelfazackerley you didn't spoil it for me! My fault.

**Comment by andybalaam** [comment]
Hints fixed in 520621268bc72e5f717eb62aaf3c0d5e87138721

**Comment by andybalaam** [comment]
And yes, I agree re-opening the individual levels is cumbersome and I should have tracked these fixes in the level set.
